### PR TITLE
refactor(backend): prefix admin rights with `ADMIN_` 

### PR DESCRIPTION
**Pull request by ulfgebhardt**
### Issues

- based on #2720 

### Todo

- [X] None

### CODE REVIEW

**Review by Mogge — COMMENTED** [review]
What is the improvement of this PR? Looks more confusing to me like this.

File: backend/src/auth/RIGHTS.ts [hunk 1]
```diff
@@ -37,21 +37,21 @@ export enum RIGHTS {
...
-  CREATION_TRANSACTION_LIST = 'CREATION_TRANSACTION_LIST',
-  LIST_TRANSACTION_LINKS_ADMIN = 'LIST_TRANSACTION_LINKS_ADMIN',
-  CREATE_CONTRIBUTION_LINK = 'CREATE_CONTRIBUTION_LINK',
-  DELETE_CONTRIBUTION_LINK = 'DELETE_CONTRIBUTION_LINK',
-  UPDATE_CONTRIBUTION_LINK = 'UPDATE_CONTRIBUTION_LINK',
+  ADMIN_LIST_UNCONFIRMED_CONTRIBUTIONS = 'LIST_UNCONFIRMED_CONTRIBUTIONS',
+  ADMIN_CONFIRM_CONTRIBUTION = 'CONFIRM_CONTRIBUTION',
+  ADMIN_SEND_ACTIVATION_EMAIL = 'SEND_ACTIVATION_EMAIL',
+  ADMIN_CREATION_TRANSACTION_LIST = 'CREATION_TRANSACTION_LIST',
+  ADMIN_LIST_TRANSACTION_LINKS_ADMIN = 'LIST_TRANSACTION_LINKS_ADMIN',
```
Comment (Mogge) [inline backend/src/auth/RIGHTS.ts:51]: Doppelt hält besser?

**Review by clauspeterhuebner — APPROVED** [review]
what does renaming of rights with the prefix ADMIN_ has to do with the event-table changes?

**Comment by ulfgebhardt** [comment]
@clauspeterhuebner Moriz is against this change - therefore we will not merge it

**Pull request closed without merging**